components/discover/Discover.tsx

```diff
@@ -0,0 +1,75 @@
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import Container from '@mui/material/Container';
+
+import DiscoverIcon from '../common/icon-discover/DiscoverIcon';
+
+import {
+  PageContainer,
+  DiscoverTitle,
+  DiscoverColumn,
+  DiscoverItemContainer,
+  DiscoverItemTitle,
+} from './styled-components';
+
+const Discover = () => {
+  return (
+    <PageContainer id="blog">
```

Comment:
Please, remove the `id` attribute. It shouldn't be blog. I am unsure how to call a link to this section

Reply:
'+